edgehog-device-runtime-containers/src/resource/container.rs

```diff
@@ -161,6 +161,7 @@ where
     }
 
     async fn delete(&mut self, ctx: &mut Context<'_, D>) -> Result<()> {
+        self.container.stop(ctx.client).await?;
```

Comment:
Is it right to remove the stop function from here?

Reply:
I've added the stop function because the backend could send a delete for a running container, and docker would error if we don't stop the container first.